docs/reference/indices/ignore-missing-component-templates.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[ignore_missing_component_templates]]
+== Config ignore_missing_component_templates
+
+The configuration option `ignore_missing_component_templates` can be used when an index template references a component template that might not exist yet. Every time a data stream is create based on the index template, the existance of the component template will be checked. If it exists, it will taken as part of the mapping. If it does not exist, it is ignored.
+
+=== Usage example
+
+In the following, one component template and an index template are created. The index template references 2 component templates but only the `@package` one exists.
+
+
+Create of the component template `logs-foo@package`. This has to be created before the index template as it is not optional:
+
+// tag::ignore_missing_component_templates[]
+
+[source,console]
+----
+PUT _component_template/logs-foo@package
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "host.name": {
+          "type": "keyword"
+        }
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+// end::ignore_missing_component_templates[]
```

Comment:
Trying to get my head around how this :end vs test continued works? What I want is that the test runs this one + executes line 52

Reply:
Unless you want to use this snippet somewhere else, this line can be removed.

Reply:
No plan to reuse any of these. I'll remove it everywhere.